### DISCUSSION TITLE
feat(Drawer): Drawer service plus the Position attribute

### DISCUSSION THREAD

**Discussion by Mydashixiong** [dotnetcore/BootstrapBlazor#4110] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the solution you'd like

Drawer service plus the Position attribute

**Comment by bb-auto** [comment]
@Mydashixiong Thank you for contacting us. We will give feedback later.

**Reply by ArgoZhang** [reply, marked as answer]
@Mydashixiong 有啊

```
public class DrawerOption
{
    public Placement Placement { get; set; } = Placement.Left;
}
```

在这里啊

**Reply by Mydashixiong** [reply]
不是 Placement ，是 Position
